pkg/kubelet/lifecycle/handlers.go

```diff
@@ -127,8 +152,16 @@ func (hr *HandlerRunner) runHTTPHandler(pod *v1.Pod, container *v1.Container, ha
 			return "", err
 		}
 	}
-	url := fmt.Sprintf("http://%s/%s", net.JoinHostPort(host, strconv.Itoa(port)), handler.HTTPGet.Path)
-	resp, err := hr.httpGetter.Get(url)
+	path := handler.HTTPGet.Path
+	url := formatURL("http", host, port, path)
```

Comment:
I think that we want to use `handler.HTTPGet.Scheme` instead of hard coding `http`.